Cargo.toml

```diff
@@ -43,7 +43,7 @@ once_cell             = { version = "^1.7.2", optional = true }
 openidconnect         = { version = "^2.0.0", optional = true, default_features = false }
 openssl               = { version = "^0.10", features = ["v110"] }
 oso                   = { version = "^0.12", optional = true, default_features = false }
-pkcs11                = { version = "^0.5", optional = true }
+cryptoki              = { version = "^0.3", optional = true, git = "https://github.com/parallaxsecond/rust-cryptoki/" }
```

Comment:
I think we should not depend on a development version - will they release 0.3 on crates.io? Or is it an option to use 0.2 instead then? This seems unstable.

Reply:
0.2 is not an option, it lacks too much (e.g. inability to login with a user pin code), I tried it first but switched to 0.3. Indeed I agree that depending on an unreleased version is not desirable. @ionut-arm: Are you able to indicate when an 0.3 release of the `cryptoki` crate can be expected?

Reply:
Yes please, v0.3.0 would be very appreciated. Potential for lots of changes before 1.0.0 is normal use of semantic versioning in Rust projects as far as we are concerned. I'd love to join the meeting but UTC 16:30 is just too late for me I'm afraid.

Reply:
In principle we would like to contribute but we also have very limited man power. If we encounter issues or need a specific feature we could look at contributing then as it would be directly aligned with our needs and goals, but at present I'm afraid you did your job too well as currently all our (and your) tests are passing ;-)

Reply:
@ionut-arm: So in the context of things we might contribute to or need, I just created https://github.com/parallaxsecond/rust-cryptoki/issues/75. However it's not urgent or perhaps even wise, but I thought it worth noting.

Reply:
> Yes please, v0.3.0 would be very appreciated. Potential for lots of changes before 1.0.0 is normal use of semantic versioning in Rust projects as far as we are concerned. 

That makes sense, thanks for confirming.

> In principle we would like to contribute but we also have very limited man power.

That's also alright, we can collaborate whenever you have any requests, like the issue you opened yesterday. Related to that, we've agreed to release the current `main`, with the caveat that we should check whether the issue you raised can be slipped in before the release, but we can continue the conversation there.

Reply:
@timbru: @ionut just did a v0.3.0 release of `cryptoki` so I've bumped our dependency to use it. Thank you VERY MUCH @ionut and team!